client/src/components/graph/drawPointsRegl.js

```diff
@@ -1,5 +1,6 @@
 // jshint esversion: 6
 const mat4 = require("gl-mat4");
+const mat3 = require("gl-vec3");
```

Comment:
shouldn't it be `const vec3 = require("gl-vec3");`?

Reply:
wow, not sure what caused that brain fault.  Fixing.